docs/content/docs/patterns/observability/logs.md

```diff
@@ -14,30 +14,46 @@ Open Service Mesh (OSM) collects logs that are sent to stdout by default. When e
+1. Find the output plugin you would like to forward your logs to in [Fluent Bit documentation](https://docs.fluentbit.io/manual/pipeline/outputs). Replace the `[OUTPUT]` section in [`fluentbit-configmap.yaml`](https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/fluentbit-configmap.yaml) with appropriate values.
 
-2. The default configuration uses CRI log format parsing. If you are using a kubernetes distribution that causes your logs to be formatted differently, you may need to add a new parser to the `[PARSER]` section and change the `parser` name in the `[INPUT]` section to one of the parsers defined [here](https://github.com/fluent/fluent-bit/blob/master/conf/parsers.conf).
+1. The default configuration uses CRI log format parsing. If you are using a kubernetes distribution that causes your logs to be formatted differently, you may need to add a new parser to the `[PARSER]` section and change the `parser` name in the `[INPUT]` section to one of the parsers defined [here](https://github.com/fluent/fluent-bit/blob/master/conf/parsers.conf).
 
-3. The logs are currently filtered to match "error" level logs and multiple filters have been used to cover differences in log formatting on various Kubernetes distros. 
-    * To change the log level being filtered on, you can update the `logLevel` value in `values.yaml` to "debug", "info", "warn", "fatal", "panic", "disabled" or "trace". 
+1. The logs are currently filtered to match "error" level logs and multiple filters have been used to cover differences in log formatting on various Kubernetes distros. 
+    * To change the log level being filtered on, you can update the `logLevel` value in [`values.yaml`](https://github.com/openservicemesh/osm/blob/main/charts/osm/values.yaml) to "debug", "info", "warn", "fatal", "panic", "disabled" or "trace". 
     * To view all logs irrespective of log level, you may remove the `[FILTER]` sections. 
+    * The modify filter has been used to add a `controller_pod_name` key/value pair to help you query logs in your output by refining results on pod name (see example usage below).
     * If you wish to apply further filtering, explore [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters).
-     
-4. Once you have updated the Fluent Bit configmap, you can deploy the sidecar during OSM installation using the `--enable-fluentbit` flag. You should now be able to interact with error logs in the output of your choice as they get generated.
 
-5. The `controller_pod_name` key/value pair has been added to the logs to help you query logs in your output by refining results on pod name (see example usage below).
+1. For these changes to take effect, run:
+    ```console
+    make build-osm
+    ```
+
+1. Once you have updated the Fluent Bit configmap, you can deploy Fluent Bit during OSM installation using:
```

Comment:
> Once you have updated the Fluent Bit configmap

Does this mean changing the configmap installed in the cluster, or modifying the template for the to-be-installed configmap?

Reply:
modifying the to-be-installed configmap. I have tried changing it with `kubectl edit` but the changes do not seem to reflect in the fluent bit results. I also think that changes at this scale could get messy with formatting and fluent bit is super sensitive to indentation being off. Does it make sense to leave it this way?

Reply:
Yeah, that makes sense. I think this might make that distinction a bit more clear:
```suggestion
1. Once you have updated the Fluent Bit configmap template, you can deploy Fluent Bit during OSM installation using:
```